openedx/course-discovery
# Django ORM POC for SFMC Catalog

[PROD-4137](https://2u-internal.atlassian.net/browse/PROD-4137)
---------
This PR adds management command to populate the course-catalog for SFMC

### Testing:

**Testing Instructions:**

1. Run the following command:
   ```bash
   ./manage.py populate_product_catalog --product_type={product_type} --output_csv={csv-file-name} --product_source={product_source} --use_gspread_client=True

Verify that the course data has been populated successfully.

- To test the gspread integration:

    - Create a service account with your Google account.
   - Add the service account credentials to the `private.py` file.
   - Create a new Google Sheet from your account and add the sheet ID to the `PRODUCT_CATALOG_SHEET_ID` variable in 
   the private.py file.
   - Run the command again to ensure the changes are reflected.
